src/preprocessing/passes/pseudo_boolean_processor.cpp

```diff
@@ -375,10 +374,10 @@ void PseudoBooleanProcessor::learnGeqSub(Node geq)
     Node y = d_pos[1];
     Node z = d_neg[0];
 
-    Node xGeq1 = mkGeqOne(x);
-    Node yGeq1 = mkGeqOne(y);
-    Node zGeq1 = mkGeqOne(z);
-    NodeManager* nm = NodeManager::currentNM();
+    Node xGeq1 = mkGeqOne(nodeManager(), x);
+    Node yGeq1 = mkGeqOne(nodeManager(), y);
+    Node zGeq1 = mkGeqOne(nodeManager(), z);
+    NodeManager* nm = nodeManager();
```

Comment:
Maybe move this up and use instead of individual `nodeManager()` calls above

Reply:
Done